src/pgwire/src/protocol.rs

```diff
@@ -1262,6 +1262,27 @@ where
             ExecuteResponse::Deallocate { all } => {
                 command_complete!("DEALLOCATE{}", if all { " ALL" } else { "" })
             }
+            ExecuteResponse::Raise { severity } => {
+                let msg = match severity {
+                    NoticeSeverity::Debug => {
+                        ErrorResponse::debug(SqlState::WARNING, "raised a test debug")
```

Comment:
Not very familiar with this area of the code, so apologies for the naive question, but the `ErrorResponse`'s text field doesn't show up anywhere; where does/should this get propagated?

Reply:
You need to `set client_min_messages to debug` before! By default, we filter out messages that are lower severity than `NOTICE` (following Postgres behavior). If you check `raise.pt`, we set it before running the tests exactly because otherwise they would be no-ops from the client-side.

Reply:
Ah, I guess I'm confused--is this message supposed to be the one we want to commit to the repo, or is it a placeholder until we implement a use of this statement?

Reply:
It's the message I want to commit, mostly because the existence of the message here is more important than the concrete message! The idea is to test how test how clients/drivers react to such messages, so the contents are more or less arbitrary.